.rhcicd/clowdapp-backend.yaml

```diff
@@ -207,6 +207,11 @@ objects:
           value: ${QUARKUS_REST_CLIENT_KC_URL}
         - name: NOTIFICATIONS_USE_SOURCES_SECRETS_BACKEND
           value: ${SOURCES_ENABLED}
+        - name: SOURCES_PSK
+          valueFrom:
+            secretKeyRef:
+              name: internal-psk
```

Comment:
Thanks for fixing the name. Now it raises another question: is `internal-psk` good enough to be put in the Notifications vault in stage and prod? From the vault alone, it will be impossible to understand that this secret is related to Sources. Of course the answer can be found in this ClowdApp template, but for a maintainability perspective this is not ideal... WDYT?

Reply:
The new name sounds better to me 👍

Ephemeral can rely on secrets from a different environment but I won't put any more details here as this is not the right place to discuss that 😄 Let's continue elsewhere.

Reply:
Perfect! After talking about it, we can consider this thread dealt with, then! Thanks :)